gammapy/utils/tests/test_time.py

```diff
@@ -0,0 +1,21 @@
+# Licensed under a 3-clause BSD style license - see LICENSE.rst
+from __future__ import (absolute_import, division, print_function,
+                        unicode_literals)
```

Comment:
Yes, please add this to any new file.

It's an Astropy coding guideline, and it also makes sense: if you don't add it, it is possible to get slightly different behaviour on Python 2 and 3 (e.g. `1/2 ==0` on Python 2 or `1/2 == 0.5` on Python 3). If test coverage is not very high and you don't put in the right test dataset, issues can go unnoticed.

I know it's annoying boilerplate code, but thinking about / discussing where you need or might need it in the future is worse than just pasting it into every file.